crux/src/Crux/Config/Common.hs

```diff
@@ -364,6 +367,10 @@ cruxOptions = Config
 
           onlineProblemFeatures <- pure noFeatures
 
+          debug <-
+            section "debug" yesOrNoSpec False
+            "Enable the Crucible debugger"
```

Comment:
It would be worth clarifying that the debugger starts before simulation begins (similarly for the other places where "Enable the Crucible debugger" is documented).